### ISSUE TITLE
is this any different at the moment than jitsi?

### ISSUE THREAD

**Issue by Fahad-pnw** [KChitnis/Nectar#1]
Hi, have you made any changes or any updates to jitsi meet? tried reviewing commit history, couldn't find much.

**Comment by m0rphtail** [comment]
> Hi, have you made any changes or any updates to jitsi meet? tried reviewing commit history, couldn't find much.

Hi, I haven't pushed them yet, but as of now it's the same as jitsi, maybe a few commits behinds.
You can help by committing, if you'd like.

**Comment by m0rphtail** [comment]
I will soon be creating some issues to start with.

**Comment by Fahad-pnw** [comment]
okay cool, what sort of issues? will you be working on streaming issue? that it only streams at YT?

**Comment by m0rphtail** [comment]
Yes, options to stream to other platforms like facebook would be nice.
